pages/style-guide.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Auto-WCAG Rules Style Guide
+---
+
+This style guide provides instructions on how to formulate and format your rules. **[Learn how to design a rule](rule-design.html)**, before starting with this guide.
```

Comment:
the `rule-design` page doesn't exist yet, but shouldn't this link point the the Markdown version so that it's navigable in the GitHub UI too?